README.md

```diff
@@ -22,60 +22,175 @@
+With Koperator, you can modify the configuration of unique Brokers, remove specific Brokers from clusters, and use multiple Persistent Volumes for each Broker.
+
+If you want to learn more about our design motivations and the scenarios that drove us to create Koperator, please continue reading on our documentation page [here](https://docs.calisti.app/sdm/koperator/scenarios/).
 
 ![Koperator architecture](docs/img/kafka-operator-arch.png)
 
->We took a different approach to what's out there - we believe for a good reason - please read on to understand more about our [design motivations](docs/features.md) and some of the [scenarios](docs/scenarios.md) which were driving us to create Koperator.
+## Quick start
 
-**Koperator** is a core part of [Banzai Cloud Supertubes](https://banzaicloud.com/products/supertubes/) (now called Cisco Streaming Data Manager) that helps you create production-ready Apache Kafka clusters on Kubernetes, with scaling, rebalancing, and alerts based self healing. While the Koperator itself is an open-source project, the [Banzai Cloud Supertubes](https://banzaicloud.com/products/supertubes/) product extends the functionality of Koperator with commercial features (for example, built-in monitoring and multiple ways of disaster recovery). Read a [detailed comparison of Supertubes and Koperator](https://banzaicloud.com/docs/supertubes/kafka-operator-supertubes-feature-comparison/).
+This quick start guide will walk you through the process of deploying Koperator on an existing Kubernetes cluster and provisioning a Kafka cluster using its custom resources.
 
-## Installation
+### Prerequisites
 
-For detailed installation instructions, see the [Banzai Cloud Documentation Page](https://banzaicloud.com/docs/supertubes/kafka-operator/install-kafka-operator/).
+To complete this guide, you will need a Kubernetes cluster (with a suggested minimum of 6 vCPUs and 8 GB RAM). You can run the cluster locally using Kind or Minikube.
 
-## Documentation
+> The quick start will help you set up a functioning Kafka cluster on Kubernetes. However, it does not include guidance on the installation of Prometheus and cert-manager, which are necessary for some of the more advanced functionality.
+
+#### Install ZooKeeper
+
+The version of Kafka that is installed by the operator requires Apache ZooKeeper. You'll need to deploy a ZooKeeper cluster if you don’t already have one.
+
+1. Install ZooKeeper using [Pravega’s Zookeeper Operator](https://github.com/pravega/zookeeper-operator).
+
+```
+helm repo add pravega https://charts.pravega.io
+helm repo update
+helm install zookeeper-operator --namespace=zookeeper --create-namespace pravega/zookeeper-operator
+```
+
+2. Create a ZooKeeper cluster.
+
+```
+kubectl create --namespace zookeeper -f - <<EOF
+apiVersion: zookeeper.pravega.io/v1beta1
+kind: ZookeeperCluster
+metadata:
+    name: zookeeper
+    namespace: zookeeper
+spec:
+    replicas: 1
+EOF
```

Comment:
**Recommended**: We **SHOULD NOT** duplicate the namespace in the command if it is part of the manifest as well, because it can be confusing if only one of those are changed on the command by the user.

Also we should explicitly specify the reclaimpolicy for the Zookeeper cluster because it is best to delete leftover metadata in most use-cases.

```suggestion
kubectl create -f - <<EOF
apiVersion: zookeeper.pravega.io/v1beta1
kind: ZookeeperCluster
metadata:
    name: zookeeper
    namespace: zookeeper
spec:
    replicas: 1
    persistence:
        reclaimPolicy: Delete
EOF
```